src/CONTRIBUTING.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Contribute to Flutter docs
+description: Types of contributors and how to contribute to the Flutter project.
+---
+
+If you would like to help contribute to the Flutter project,
+we’re happy to have your help! Anyone can contribute, whether you’re new to
+the project or you’ve been around a long time, and whether you self-identify
+as a developer, an end user, or someone who just can’t stand seeing typos.
+
+For more ways to get involved in the Flutter community or to learn about us,
+also visit the [Flutter community](/community) page.
+
+Looking for the [Effective Dart]({{site.www}}/guides/language/effective-dart)?
+This comprehensive set of guides for the Dart language (which is used by Flutter),
+includes guides on style, documentation, usage, and design.
+
+* [Types of contributor](#types-of-contributor)
+* [Ways to contribute](#ways-to-contribute)
+
+# Types of contributor
+
+You can contribute to the Flutter SDK or the Flutter website. There are two
+levels of contributors:
+
+* A _contributor_ has signed the Contributor License Agreement (CLA) and
+  submittecd one or more patches.
+
+* A _trusted contributor_ is granted commit access by gaining our trust and
+  demonstrating a commitment to Flutter.
```

Comment:
I don't want to draw attention to this, it feels elitist/classist. There's also a lot more to contributing code than just this, for example, you have to read the contributing doc, which walks you through setting up your environment, etc, you have to read the style guide and design principles doc, you have to interact within the bounds of our code of conduct, etc. Plus, there's plenty of ways to contribute to the community that aren't covered by these "types", as you describe below.